ProjFS.Mac/PrjFSKext/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -56,18 +119,17 @@ kern_return_t VirtualizationRoots_Cleanup()
     return KERN_FAILURE;
 }
 
-VirtualizationRoot* VirtualizationRoots_FindForVnode(vnode_t vnode)
+VirtualizationRootHandle VirtualizationRoot_FindForVnode(vnode_t vnode)
 {
-    VirtualizationRoot* root = nullptr;
+    VirtualizationRootHandle rootIndex = RootHandle_None;
     
     vnode_get(vnode);
     // Search up the tree until we hit a known virtualization root or THE root of the file system
-    while (nullptr == root && NULLVP != vnode && !vnode_isvroot(vnode))
+    while (rootIndex == RootHandle_None && NULLVP != vnode && !vnode_isvroot(vnode))
```

Comment:
@sanoursa, we have a mix right now with our checks, sometimes it's `VALUE == var` and othertimes it's `var == VALUE`.  Did we decide on preferring one over the other, and what is the rule for `!=`?

Reply:
(No need to clean this up in this PR, I'm just wondering what our style guideline is for this)

Reply:
I definitely prefer yoda-speak in C/C++ code to eliminate any risk of accidental assignments, but we have not enforced it consistently.